DSCResources/MSFT_xIisFeatureDelegation/MSFT_xIisFeatureDelegation.schema.mof

```diff
@@ -0,0 +1,6 @@
+[ClassVersion("1.0.0"), FriendlyName("xIisFeatureDelegation")] 
+class MSFT_xIisFeatureDelegation : OMI_BaseResource
+{
+  [Key] string SectionName;
+  [Key,ValueMap{"Allow", "Deny"},Values{"Allow", "Deny"}] string OverrideMode;
+};
```

Comment:
We can file a feature request on that, but I think its more of how the PowerShell team wants to mange thier stuff in GitHub. :grin: